src/cmlibs/exporter/webgl.py

```diff
@@ -94,11 +94,14 @@ def _define_settings_obj(self):
         return settings_obj
 
     def export_webgl(self):
+        scene = self._document.getRootRegion().getZincRegion().getScene()
+        self.export_webgl_from_scene(scene)
+
+    def export_webgl_from_scene(self, scene, scene_filter=None):
         """
         Export graphics into JSON format, one json export represents one
         surface graphics.
         """
```

Comment:
This documentation doesn't belong with this method.  And it should have its own documentation, nothing major but should include information on the arguments.